.github/workflows/publish.yml

```
@@ -1,6 +1,9 @@
 name: Plugins - CD
 run-name: Deploy ${{ inputs.branch }} to ${{ inputs.environment }} by @${{ github.actor }}
 
+permissions:
+  contents: read
```

We actually also need `id-token: write` to access Vault

https://github.com/grafana/shared-workflows/blob/c0944351c68e107649cb9c9a86204b400b607564/actions/get-vault-secrets/README.md?plain=1#L20-L23

```suggestion
permissions:
  contents: read
  id-token: write
```